gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveDatasetFinder.java

```diff
@@ -113,6 +117,12 @@ protected HiveDatasetFinder(FileSystem fs, Properties properties, HiveMetastoreC
 
   protected HiveDatasetFinder(FileSystem fs, Properties properties, HiveMetastoreClientPool clientPool,
       EventSubmitter eventSubmitter) throws IOException {
+    this(fs, properties, clientPool, eventSubmitter, ConfigClientCache.getClient(VersionStabilityPolicy.STRONG_LOCAL_STABILITY));
```

Comment:
What does `ConfigClientCache.getClient(VersionStabilityPolicy.STRONG_LOCAL_STABILITY)` return if there is no configstore configured?